types/types.go

```diff
@@ -256,7 +256,10 @@ type Info struct {
 	Runtimes           map[string]Runtime
 	DefaultRuntime     string
 	Swarm              swarm.Info
-	LiveRestore        bool
+	// Determine whether containers should be kept running when the
```

Comment:
No. The entire standard library and all the code in docker follows (or should) this standard.

See https://github.com/golang/go/wiki/CodeReviewComments#doc-comments